beacon-chain/blockchain/head.go

```diff
@@ -290,3 +310,53 @@ func (s *Service) getJustifiedBalances() []uint64 {
 	defer s.justifiedBalancesLock.RUnlock()
 	return s.justifiedBalances
 }
+
+// Notifies a common event feed of a new chain head event. Called right after a new
+// chain head is determined, set, and saved to disk.
+func (s *Service) notifyNewHeadEvent(
+	newHeadSlot types.Slot,
```

Comment:
Optional feedback:
These arguments could be cleaner and reduced...  (ie. get slot from block, get state root from block... etc)

Reply:
I can fix in the next PR which integrates this one